helm/tempo/templates/ingress.yaml

```diff
@@ -0,0 +1,71 @@
+{{- if .Values.ingress.enabled -}}
+# Ingress resource to send traces to Tempo over HTTP
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    {{- with .Values.ingress.http.annotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ include "tempo.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "tempo.labels" . | nindent 4 }}
+spec:
+  {{- with .Values.ingress.className }}
```

Comment:
I know this is counter intuitive but most upstream chart use ingress.ingressClassName

Reply:
renamed that to ingressClassName